### ISSUE TITLE
Proposal: Getting Started Guide

### ISSUE THREAD

**Issue by digisavvy** [wprig/wprig#236]
<img align="right" width="90" height="90"
     src="https://avatars1.githubusercontent.com/u/38340689"
     title="WP Rig logo by Morten Rand-Hendriksen">
# WP Rig: Getting Started Guide

## Getting Started
So you want to level up your theme development game by using a truly progressive, accessible, forward-thinking WordPress theme. Great! You've downloaded your theme, installed the requisite npm packages and you're ready to get going! You're filled with excitement! But what do you do next? We're glad you asked! This little guide will get you up and running right quickly!

### Folder Structure
- TODO - Tree view of the folder structure
- TODO - Discuss what files should be ignored/included

### Directory Structure
- TODO - Explain what files can be edited and what files should *not* be edited.

#### Compiling Assets
- TODO - Briefly discuss BrowserSync, what it does and what needs to be done to start getting your assets to refresh via BrowserSync.
- TODO - Explain how compiling works, what the asset build pipeline looks like and where compiled assets are compiled to.
- TODO - Explain how to add custom CSS to your theme.

#### Getting Ready for Production
- TODO - Explain how prod ready assets are created, where they're sent and what theme should be used.

#### Explain Theme Development Recommended Practices with WPRig. 
- TODO - Explain where one should edit CSS and js files

### Example Quick Start
- TODO - Explain what to edit in `themeConfig.js` file.
- TODO - Explain where to create Sass folder and files. Provide basic Sass structures.
- TODO - Explain how to either enqueue your stylesheet or how to add it the `style/Component.php` file.
- TODO - Discuss how to add and modify Google Fonts within the theme.

**Comment by mor10** [comment]
Yes to this! @digisavvy do you want to expand on this and make it part of either the wiki here in the repo or a guide on wprig.io?

**Comment by digisavvy** [comment]
I'd be happy to expand on this. I think a Wiki Entry would probably be best place for it. Getting started or Readme guides can reference it.
